docs-java/faq.mdx

```diff
@@ -37,6 +37,29 @@ OpenAiClient client2 = OpenAiClient.withCustomDestination(destination);
 
 For more information, please refer to the [AI Core connectivity guide](connecting-to-ai-core) and the [SAP Cloud SDK documentation](https://sap.github.io/cloud-sdk/docs/java/features/connectivity/http-destinations).
 
+### Can I apply a minor version update for the SAP Cloud SDK for AI dependency?
+
+Yes, we recommend updating to the latest version of the SAP Cloud SDK for AI.
+The SAP Cloud SDK for AI follows [semantic versioning](https://semver.org) so minor version updates should generally be safe and backward-compatible.
+However, there are some exceptional cases where API changes may occur:
+
+#### Internal Classes
+
+Internal classes are intended for use only within the SAP Cloud SDK for AI itself.
+Their API signatures may change without notice and are not guaranteed to be stable.
+
+- [Orchestration models](https://github.com/SAP/ai-sdk-java/tree/main/orchestration/src/main/java/com/sap/ai/sdk/orchestration/model)
+- [OpenAI models](https://github.com/SAP/ai-sdk-java/tree/main/foundation-models/openai/src/main/java/com/sap/ai/sdk/foundationmodels/openai/generated/model)
+
+#### Generated Classes
+
+The following clients are primarily generated by the [SAP Cloud SDK Generator](https://sap.github.io/cloud-sdk/).
+Their API signatures may change, for example, due to updates in the underlying specifications:
+
+- [AI Core client](https://github.com/SAP/ai-sdk-java/tree/main/core/src/main/java/com/sap/ai/sdk/core)
```

Comment:
```suggestion
- [AI Core client](https://github.com/SAP/ai-sdk-java/tree/main/core/src/main/java/com/sap/ai/sdk/core/client)
```

Reply:
If we want to make a more granular list, then we should put 2 items, right?
- https://github.com/SAP/ai-sdk-java/tree/main/core/src/main/java/com/sap/ai/sdk/core/client
- https://github.com/SAP/ai-sdk-java/tree/main/core/src/main/java/com/sap/ai/sdk/core/model

Reply:
Yes